packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.test.tsx

```diff
@@ -0,0 +1,146 @@
+  RecentPublicationsGridProps,
+} from './RecentPublicationsGrid'
+import { createWrapper } from '../../testutils/TestingLibraryUtils'
+import useGetQueryResultBundle from '../../synapse-queries/entity/useGetQueryResultBundle'
+import { ColumnTypeEnum } from '@sage-bionetworks/synapse-types'
+
+jest.mock('../../synapse-queries/entity/useGetQueryResultBundle')
+const mockUseGetQueryResultBundle = jest.mocked(useGetQueryResultBundle)
+
+describe('RecentPublicationsGrid Tests', () => {
+  const mockProps: RecentPublicationsGridProps = {
+    sql: 'SELECT * FROM syn123',
+    buttonLink: 'https://example.com',
+    buttonLinkText: 'View All Publications',
+    summaryText: 'This is a summary.',
+  }
+
+  const mockQueryResult = {
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryResultBundle',
+    queryResult: {
+      concreteType: 'org.sagebionetworks.repo.model.table.QueryResult',
+      queryResults: {
+        concreteType: 'org.sagebionetworks.repo.model.table.RowSet',
+        tableId: 'syn64042839',
+        etag: 'DEFAULT',
+        headers: [
+          {
+            name: 'id',
+            columnType: ColumnTypeEnum.ENTITYID,
+            id: '81721',
+          },
+          {
+            name: 'Tag',
+            columnType: ColumnTypeEnum.STRING_LIST,
+            id: '81722',
+          },
+          {
+            name: 'Journal',
+            columnType: ColumnTypeEnum.STRING,
+            id: '81723',
+          },
+          {
+            name: 'Title',
+            columnType: ColumnTypeEnum.STRING,
+            id: '81724',
+          },
+          {
+            name: 'PublicationDate',
+            columnType: ColumnTypeEnum.DATE,
+            id: '81725',
+          },
+        ],
+        rows: [
+          {
+            rowId: 1,
+            values: [
+              '1',
+              '["Tag1_1", "Tag1_2"]',
+              'Journal1',
+              'Title1',
+              '1725819400000',
+            ],
+          },
+          {
+            rowId: 2,
+            values: [
+              '2',
+              '["Tag2_1", "Tag2_2"]',
+              'Journal2',
+              'Title2',
+              '1709578435000',
+            ],
+          },
+        ],
+      },
+    },
+    selectColumns: [
+      {
+        name: 'id',
+        columnType: ColumnTypeEnum.ENTITYID,
+        id: '81721',
+      },
+      {
+        name: 'Tag',
+        columnType: ColumnTypeEnum.STRING_LIST,
+        id: '81722',
+      },
+      {
+        name: 'Journal',
+        columnType: ColumnTypeEnum.STRING,
+        id: '81723',
+      },
+      {
+        name: 'Title',
+        columnType: ColumnTypeEnum.STRING,
+        id: '81724',
+      },
+      {
+        name: 'publicationDate',
+        columnType: ColumnTypeEnum.DATE,
+        id: '81725',
+      },
+    ],
+  }
+
+  beforeEach(() => {
+    jest.clearAllMocks()
+    mockUseGetQueryResultBundle.mockReturnValue({
+      data: mockQueryResult,
+    } as any)
```

Comment:
Recommend creating the return value with `getUseQuerySuccessMock` in ReactQueryMockUtils.ts, which will give you fields beyond just `data`